app/docs/0.11.x/configuration.md

```diff
@@ -720,6 +726,62 @@ in memory. To read the file again after modifying it, Kong must be reloaded.
 
 Default: `/etc/hosts`
 
+---
+
+##### **dns_order**
+
+The order in which to resolve different
+record types. The `LAST` type means the
+type of the last successful lookup (for the
+specified name). The format is a (case
+insensitive) comma separated list.
+
+Default: `LAST,SRV,A,CNAME`
+
+---
+
+##### **dns_stale_ttl**
+
+Once a record expires, how much longer (in
+seconds) may the current stale record be 
+used while the refresh query is executed in
+the background.
```

Comment:
This sentence was updated in the current PR, and the change should be reflected here as well